monai/utils/module.py

```diff
@@ -450,7 +453,47 @@ def _try_cast(val: str):
     return True
 
 
-try:
-    PT_BEFORE_1_7 = torch.__version__ != "1.7.0" and version_leq(torch.__version__, "1.7.0")
-except (AttributeError, TypeError):
-    PT_BEFORE_1_7 = True
+def pytorch_after(major, minor, patch=0, current_ver_string=None) -> bool:
+    """
+    Compute whether the current pytorch version is after or equal to the specified version.
+    The current system pytorch version is determined by `torch.__version__` or
+    via system environment variable `PYTORCH_VER`.
+
+    Args:
+        major: major version number to be compared with
+        minor: minor version number to be compared with
+        patch: patch version number to be compared with
+        current_ver_string: if None, `torch.__version__` will be used.
+
+    Returns:
+        True if the current pytorch version is greater than or equal to the specified version.
+    """
+    try:
+        if current_ver_string is None:
+            _env_var = os.environ.get("PYTORCH_VER", "")
+            current_ver_string = _env_var if _env_var else torch.__version__
+        parts = f"{current_ver_string}".split("+", 1)[0].split(".", 3)
+        while len(parts) < 3:
+            parts += ["0"]
+        c_major, c_minor, c_patch = parts[:3]
+    except (AttributeError, ValueError, TypeError):
+        c_major, c_minor = get_torch_version_tuple()
+        c_patch = "0"
+    c_mn = int(c_major), int(c_minor)
+    mn = int(major), int(minor)
+    if c_mn != mn:
+        return c_mn > mn
```

Comment:
Would it make sense to use [version_leq](https://github.com/Project-MONAI/MONAI/blob/f54b7af0647dc5a1ea259765a19d3d3bcb3c42c5/monai/utils/module.py#L410) in this context? Thinking about this and `version_leq` in general maybe we should be using [packaging.version](https://packaging.pypa.io/en/latest/version.html) if we could assume `packaging` is always going to be installed, it's third party but it would be an odd environment that didn't have it.

Reply:
the third party one is not included in some docker images...I added your suggestion as the preferred solution.